content/source/docs/cloud/overview.html.md

```diff
@@ -92,15 +92,15 @@ The remote backend also supports state manipulation commands like `terraform imp
 
 - More info: [CLI-driven Runs](./run/cli.html)
 
-### Private Module Registry
+### Private Registry
 
 Even small teams can benefit greatly by codifying commonly used infrastructure patterns into reusable [modules][].
 
-Terraform CLI can already fetch modules from arbitrary VCS sources, but Terraform Cloud improves this with a private module registry. Users throughout your organization can browse a directory of internal modules, and can specify flexible version constraints for the modules they use in their configurations. Easy versioning lets downstream teams use modules with confidence, and frees upstream teams to iterate faster.
+Terraform CLI can already fetch modules from arbitrary VCS sources, but Terraform Cloud improves this with a private registry. Users throughout your organization can browse a directory of internal providers and modules, and can specify flexible version constraints for the modules they use in their configurations. Easy versioning lets downstream teams use private modules with confidence, and frees upstream teams to iterate faster.
```

Comment:
So, both modules and providers can be fetched from arbitrary _sources_. Perhaps we can say:
Terraform can fetch providers and modules from many sources. Terraform Cloud makes it easier to find providers and modules to use with a private registry." and then the rest is fine.

Reply:
done! Thank you!